lib/internal/http2/core.js

```diff
@@ -394,9 +394,11 @@ function onSessionHeaders(handle, id, cat, flags, headers, sensitiveHeaders) {
       }
     } else if (cat === NGHTTP2_HCAT_PUSH_RESPONSE) {
       event = 'push';
-      // cat === NGHTTP2_HCAT_HEADERS:
     } else if (!endOfStream && status !== undefined && status >= 200) {
       event = 'response';
+    } else if (endOfStream && status !== undefined && status >= 200) {
+      // cat === NGHTTP2_HCAT_HEADERS
+      event = 'response';
```

Comment:
maybe I'm missing something, but isn't this just `if (status !== undefined && status >= 200)`?

Reply:
It is

Reply:
I just unified them